alfasim_sdk_api/common.h

```diff
@@ -31,6 +31,17 @@ enum TimestepScope
     PREVIOUS=1
 };
 
+enum StateVariable {
```

Comment:
I don't know if here is the correct place, but I think would be nice add a description to each of StateVariables, just to let clear what each one represents.

Reply:
done